apps/judicial-system/web/src/routes/Judge/HearingArrangements/HearingArrangements.tsx

```diff
@@ -100,34 +100,40 @@ export const HearingArrangements: React.FC = () => {
   }, [])
 
   useEffect(() => {
-    if (data && workingCase === null) {
+    if (data && !workingCase) {
       let theCase = data.case
 
       if (!theCase.courtDate) {
         theCase = { ...theCase, courtDate: theCase.requestedCourtDate }
 
-        updateCase(
-          theCase.id,
-          parseString('courtDate', theCase.requestedCourtDate),
-        )
+        if (theCase.requestedCourtDate) {
```

Comment:
Væri ekki betra að setja þetta if með if (!theCase.courtDate)?

Reply:
Ég hugsaði það líka fyrst en ef ég geri það þá fæ ég TypeScript villu þegar ég nota strict.. Ég er náttúrulega að breyta `theCase.requestedCourtDate` áður en ég geri updateCase þannig það meikar sens að þurfa að verja mig áður.. Ég gæti verið að breyta `theCase.requestedCourtDate` í undefined t.d. 🤷‍♂️

![Screenshot 2020-11-13 at 08 40 58](https://user-images.githubusercontent.com/3789875/99047324-f66db980-258b-11eb-8eeb-3ac1554da18b.png)

Reply:
getur svindlað og sett courtData þarna inn? er það of ljótt?

Reply:
Fylgi ekki alveg.. Setja courtDate inn hvar?

Reply:
Done :D